CHANGELOG.md

```diff
@@ -6,6 +6,20 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 Currently, this repo is in Prerelease. When it is released, this project will adhere to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 ========
 
+## [0.5.0]
+
+### Breaking Changes
+
+-   `EditionCard` is now `Card`
+
+### Adds
+
+-   Icon for audiobook
+
+### Changes
+
+-   Buttons from black by default to blue
```

Comment:
this is probably breaking too, since I think RN will now have to override buttons styles.

Reply:
@hokei Let's follow up with Ellen on this one—I'm fairly certain this is the new intended style for ResearchNow. If that isn't being reflected on your template pages in Metronome than we should get those updated.

(I mean I'm happy to mark it as a breaking change, I'm just saying that it _shouldn't_ be for RN.)